### PR TITLE
Fixes every bug in the codebase.

### PR DESCRIPTION
**Pull request by Manly-Dwarf**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

## Changelog
:cl: Manly Dwarf
del: Due to balance concerns, everything in the entire codebase has been deleted.
/:cl:

## About The Pull Request
oy you /tg/station SHILLS think you can out-do every other server on removal PRs?
think again!

## Why It's Good For The Game
fixes all of our concerns with the codebase in one fell swoop

obviously a speedmerge candidate

### CODE REVIEW

**Comment by ghost** [comment]
+1 end it all.

**Comment by GuyonBroadway** [comment]
Not modular.

**Comment by Manly-Dwarf** [comment]
WTF

**Comment by GuyonBroadway** [comment]
Modularize the fix

**Comment by CasualStation** [comment]
+1 perfectly balanced

**Comment by Majick1** [comment]
+1 perfectly balanced

File: code/__DEFINES/radio.dm [hunk 1]
```diff
@@ -1,115 +0,0 @@
-// Radios use a large variety of predefined frequencies.
```
Comment (GuyonBroadway) [inline code/__DEFINES/radio.dm:1]: Modularise dis

**Review by GuyonBroadway — DISMISSED** [review]

**Comment by ExpAntique** [comment]
+1 it’s all so tiring

**Review by ghost — APPROVED** [review]
Code looks ready to go

**Comment by Manly-Dwarf** [comment]
so many conflicts why

**Pull request closed without merging**